pkg/s3wrapper/client.go

```diff
@@ -32,6 +32,7 @@ const awsEndpointSuffix = ".amazonaws.com"
 
 // TODO: Can we use RHCOS_VERSION or BASE_OS_IMAGE in the Makefile to set the value for
 // BaseObjectName?
+const RHCOSBaseURL = "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.6/46.82.202009222340-0/x86_64/rhcos-46.82.202009222340-0-live.x86_64.iso"
```

Comment:
Where was this value coming from before?
I mean, I can tell it was part of the image build, but should this be in configuration somewhere rather than hardcoded?

Reply:
Oh it's in the Makefile and now copied here ... Can we deduplicate this somehow?

Reply:
Deduped it by taking it out of the Makefile.